src/index.css

```diff
@@ -21,12 +21,27 @@ h2 {
 
 
 /* article card */
-
 .article {
   text-align: center;
   padding-bottom: 4em;
 }
 
 .article-title{
   font-size: 1.5em;
+}
+
```

Comment:
At the moment this app isn't responsive so the view isn't readable when in mobile view. As we recommend designing for mobile first, all of our tutors will be expecting this app to work in mobile view. Please fix the responsiveness and design for mobile first moving forward